Content.Server/GameTicking/Rules/RevolutionaryRuleSystem.cs

```diff
@@ -110,9 +110,19 @@ private void OnRoundEndText(RoundEndTextAppendEvent ev)
             ev.AddLine(Loc.GetString("head-rev-initial-count", ("initialCount", headrev.HeadRevs.Count)));
             foreach (var player in headrev.HeadRevs)
             {
-                ev.AddLine(Loc.GetString("head-rev-initial",
+                if (_mindSystem.TryGetSession(player.Value, out var session))
+                {
+                    var username = session.Name;
+                    ev.AddLine(Loc.GetString("head-rev-initial",
+                    ("name", player.Key),
+                    ("username", username)));
+                }
+                else
+                {
+                    ev.AddLine(Loc.GetString("head-rev-initial",
                     ("name", player.Key),
                     ("username", player.Value)));
```

Comment:
i think this will break, have a separate locale id for no username, like objectives roundend text does